CovidTracking/CovidTracking/Uitils/BindableType.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  BindableType.swift
+//  CovidTracking
+//
+//  Created by Quang Nguyen on 5/19/21.
+//  Copyright © 2021 Tiến. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol BindableType {
+    associatedtype ViewModelType
+    
+    var viewModel: ViewModelType! { get set }
+    
+    func bindViewModel()
+}
+
+extension BindableType where Self: UIViewController {
+    mutating func bind(to viewModel: Self.ViewModelType) {
+        self.viewModel = viewModel
+        loadViewIfNeeded()
+        bindViewModel()
+    }
+}
+
+extension BindableType where Self: UIView {
```

Comment:
view thì cần gì bindable nhỉ?

Reply:
e có dùng cell.bind(to:...) nên cần dùng ạ